charts/perses/templates/statefulset.yaml

```diff
@@ -26,6 +26,10 @@ spec:
         {{- end }}
     spec:
       serviceAccountName: {{ include "perses.serviceAccountName" . }}
+      {{- if .Values.persistence.enabled }}
```

Comment:
We should allow users set security context even if they decide to not use a Persistent Volume.

Reply:
Great point. fixed in a5ebe37990b9c85bddc0f76ffe8c486e41165873 commit